tests/macsec/macsec_config_helper.py

```diff
@@ -76,22 +76,33 @@ def enable_macsec_port(host, port, profile_name):
             lines=['mac security profile {}'.format(profile_name)],
             parents=['interface {}'.format(port)])
         return
+    
+    pc = find_portchannel_from_member(port, get_portchannel(host))
+    if pc:
+        host.command("sudo config portchannel {} member del {} {}".format(getns_prefix(host, port), pc["name"], port))
```

Comment:
Should we wait few seconds at here to guarantee that members have been totally removed from the portchannel? Otherwise the next command `sonic-db-cli {} CONFIG_DB HSET 'PORT|{}' 'macsec' '{}'` may cause to enable MACsec on a portchannel member. I think there is no any sync between these two commands.

Reply:
Taken care of Ze, I have added a small delay here. I was earlier doing it in test_portchanel